### ISSUE TITLE
Without old lighting, using any shader will make "big" blocks very dark in some areas

### ISSUE THREAD

**Issue by noacubestudio** [sp614x/optifine#2494]
Tried a few different shaders, this also happens in plenty Optifine versions. When a block is bigger than the 1x1x1 area, I think.

![grafik](https://user-images.githubusercontent.com/31667392/59553190-ce8eb680-8f91-11e9-9809-3c6b6a376ae7.png)

**Comment by sp614x** [comment]
The old lighting should be set to Default so the shader pack can pick the correct option.

**Comment by noacubestudio** [comment]
Default will be "OFF''. So it will have these weird dark areas. Every shader pack does that, so it looks like the only option is to use the old (otherwise worse?) lighting?

**Comment by noacubestudio** [comment]
Looks like it's also related to the Smooth Lighting level, which makes sense. (0 = it looks fine)

**Comment by sp614x** [comment]
OFF means that the user wants the old lighting to be OFF.
Default means that the shader pack can decide ON/OFF depending on how it works.

**Comment by noacubestudio** [comment]
Yes, I was just wondering why the 'new' lighting always has this problem regardless of the shader.
